forge-gui/res/cardsfolder/upcoming/conduit_of_worlds.txt

```diff
@@ -0,0 +1,11 @@
+Name:Conduit of Worlds
+ManaCost:2 G G
+Types:Artifact
+S:Mode$ Continuous | Affected$ Land.YouOwn | MayPlay$ True | AffectedZone$ Graveyard | Description$ You may play lands from your graveyard.
+A:AB$ Play | Cost$ T | Defined$ Targeted | ValidSA$ Spell | SorcerySpeed$ True | TgtPrompt$ Choose a creature card in your graveyard | CheckSVar$ X | SVarCompare$ EQ0 | TgtZone$ Graveyard | ValidTgts$ Permanent.nonLand+YouOwn | RememberPlayed$ True | Optional$ True | SubAbility$ DBEffect |SpellDescription$ Choose target nonland permanent card in your graveyard. If you haven't cast a spell this turn, you may cast that card. If you do, you can't cast additional spells this turn. Activate only as a sorcery.
```

Comment:
`| Defined$ Targeted` probably isn't needed? shouldn't it already use ValidTgts?